code/datums/keybinding/human.dm

```diff
@@ -1,65 +1,76 @@
 /datum/keybinding/human
 	category = CATEGORY_HUMAN
 	weight = WEIGHT_MOB
-
+	description = ""
 
 /datum/keybinding/human/quick_equip
-	hotkey_keys = list("E")
-	name = "quick_equip"
-	full_name = "Quick equip"
-	description = ""
 	keybind_signal = COMSIG_KB_QUICKEQUIP
+	///The quick equip slot the keybind will equip to, checked by the reciever of the keybind signal.
+	var/quick_equip_slot
+
+/datum/keybinding/human/quick_equip/quick_equip_1
+	hotkey_keys = list("E")
+	name = "quick_equip_1"
+	full_name = "Quick equip 1"
+	quick_equip_slot = 1
 
```

Comment:
You can just use a macro here to minimize boilerplate
Also no numbers in typepaths